auth/app/src/main/java/com/google/firebase/quickstart/auth/kotlin/PhoneAuthActivity.kt

```diff
@@ -57,6 +58,8 @@ class PhoneAuthActivity : Activity() {
                     // Invalid request
                 } else if (e is FirebaseTooManyRequestsException) {
                     // The SMS quota for the project has been exceeded
+                } else if (e instanceof FirebaseAuthMissingActivityRecaptchaException) {
+                    // Recaptcha verification attempted with invalid acitvity
```

Comment:
Maybe say "with null activity"?
```suggestion
                    // reCAPTCHA verification attempted with null Activity
```